docker-compose.dev.yml

```diff
@@ -12,3 +12,10 @@ services:
       timeout: 3s
       retries: 30
       start_period: 20s
+  navigator-search-indexer:
+    build:
+      context: .
+    entrypoint: []
+    environment:
```

Comment:
Do we want a health check on this for completeness?  Or is there other code that checks if its "up" ?

Reply:
All good. Its not a service, and is run as a job. So it's never actually 'up'. Just triggered to run the tests and this sets up the config for running it